src/main/scala/org/clulab/wm/eidos/apps/GroundCanonicalNames.scala

```diff
@@ -40,6 +42,24 @@ object GroundCanonicalNames extends App {
         (name, value, nameToIsLeaf(name))
       }
     }
+
+    def ground(text: String, canonicalName: String): Option[(String, Float, Boolean)] = {
+      val canonicalNameParts = canonicalName.split(' ')
+      val start = text.toLowerCase.indexOf(canonicalNameParts.head.toLowerCase)
+      val stop = text.toLowerCase.indexOf(canonicalNameParts.last.toLowerCase, start)
```

Comment:
These seem problematic -- what if we had the last word appearing N times in a sentence, isn't `indexOf` going to return the first occurrence?